Dockerfile

```diff
@@ -70,7 +67,7 @@ RUN set -x \
   && yarn test \
 # Actually build the project.
   && yarn build:clean \
-  && yarn build \
+  && NODE_ENV=production yarn build \
```

Comment:
But I still want to run the build in production mode.

Reply:
Yeah, I think it's still good to have just in case